WireUI/Sources/WireConversationUI/ConverationTitleView.swift

```diff
@@ -0,0 +1,120 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+import WireAccountImageUI
+import WireDesign
+import WireReusableUIComponents
+
+public class ConversationTitleView: UIView {
+
+    private let source: ConversationTitleSource
+
+    private let accountImageView = AccountImageView()
+    private let nameLabel = UILabel()
+    private let subtitleLabel = UILabel()
+    private let dropdownImage = UIImageView(image: .dropdown)
+
+    init(source: ConversationTitleSource) {
+        self.source = source
+        super.init(frame: CGRect.zero)
+        configureViews()
+        configureLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureViews() {
+
+        nameLabel.font = FontSpec.normalSemiboldFont.font
+        nameLabel.textColor = SemanticColors.Label.textDefault
+        nameLabel.text = source.title
+
+        subtitleLabel.font = FontSpec.smallBoldFont.font
```

Comment:
suggestion: I think we should specify fonts like this, but I'm not 100% sure for navigation bar views.
Same style for the `nameLabel`.
```suggestion
        subtitleLabel.font = .preferredFont(forTextStyle: .headline)
```

Reply:
I'll update fonts in one of next PRs if that's ok